### DISCUSSION TITLE
請問該如何跨電腦同步詞庫

### DISCUSSION THREAD

**Discussion by larryjclai** [vChewing/vChewing-macOS#480] [Q&A]
請問如果想要在不同的電腦煎同步詞庫該如何實現？
現在工作用的電腦和自家用的電腦都是使用威注音，但兩邊詞庫不同步比較困擾

**Comment by ShikiSuen** [comment]
@larryjclai 我一般都是用 iCloud 同步詞庫。但您剛才也看到了輸入法官方推文提到這個行為在 macOS 14 系統下可能會出現的問題。
解決方式是用 iCloud Container，但這會導致輸入法必須只能由專案主人的 developer program 簽證建置，妨礙該輸入法的開源流通性。
我在想一個備選方案，抑或是我接下來一兩週會向 Apple Developer 開 DTK 工單求援。
